.travis.yml

```diff
@@ -79,7 +79,7 @@ before_script:
   - elapsed "before_script"
   # Do a fail-fast check for Python syntax errors or undefined names.
   # Use the comment '# noqa: <error code>' to suppress.
-  - flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics
+  - flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
```

Comment:
Also, a link to <http://flake8.pycqa.org/en/3.7.8/user/error-codes.html>
in a comment would be much appreciated here.